app/scripts/modules/amazon/src/loadBalancer/configure/common/targetGroupValidators.spec.ts

```diff
@@ -0,0 +1,162 @@
+import * as Validators from './targetGroupValidators';
+
+const mockTargetGroup = {
+  attributes: {
+    deregistrationDelay: 600,
+    stickinessDuration: 8400,
+    stickinessEnabled: false,
+    stickinessType: 'lb_cookie',
+  },
+  healthCheckInterval: 10,
+  healthCheckPath: '/healthcheck',
+  healthCheckPort: 7001,
+  healthCheckProtocol: 'HTTP',
+  healthCheckTimeout: 5,
+  healthyThreshold: 10,
+  name: 'targetgroup',
+  port: 7001,
+  protocol: 'HTTP',
+  targetType: 'ip',
+  unhealthyThreshold: 2,
+  account: 'test',
+  cloudProvider: 'aws',
+  healthTimeout: 1000,
+  healthInterval: 10,
+  loadBalancerNames: ['loadbalancer1', 'loadbalancer2'],
+  region: 'us-east-1',
+  type: '',
+};
+
+describe('Target Group validators', () => {
+  describe('of used names', () => {
+    it('returns an error if the name exists already', () => {
+      const existingGroups = {
+        test: {
+          'us-east-1': ['targetgroup'],
+        },
+      };
+      const actual = Validators.isNameInUse(existingGroups, 'test', 'us-east-1')(mockTargetGroup.name);
+      const expected = 'There is already a target group in test:us-east-1 with that name.';
```

Comment:
is there value in asserting the exact error message in these tests (given that they'll likely just get copy-pasted across files when changed), as opposed to only asserting that the validator returns _some_ error message when it's expected to? e.g. `expect(isString(actual))`?

Reply:
Its more of a nit-picky value to me. It brings value if we care about the specific language being used. I come from a world of very strict language requirements, therefore I have a habit of this, but I am happy to change this if you feel it is too strict.